### ISSUE TITLE
[BUG] ARC Tool issue - Aria attribute is not allowed 

### ISSUE THREAD

**Issue by Satyam15302** [UKHO/admiralty-design-system#211]
## Describe the bug
 
The `aria-label` attribute is being used on elements with the `generic` role, which is not allowed. This violates WCAG 2.2 Success Criterion 4.1.2 (Name, Role, Value).
 
## To Reproduce
 
 
1. Open the web application.

2. Navigate to the "Confirm exchange set content" page or "Selecting ENCs" page.

3. Use a web accessibility testing tool (e.g., ARC Toolkit).

4. Observe the error regarding the use of `aria-label` on `generic` role elements.
 
## Expected behavior
 
The `aria-label` attribute should not be used on elements with the `generic` role. Instead, elements should use roles that allow the `aria-label` attribute, such as `button`, `checkbox`, etc.
 
## Screenshots
 
![ARIA attribute is not allowed](https://github.com/UKHO/admiralty-design-system/assets/108277912/e5aa26de-6ab0-454a-a42c-0a1a86395162)
![ARIA attribute is not allowed-File Upload](https://github.com/UKHO/admiralty-design-system/assets/108277912/26f34994-3f56-4f7c-9395-ac56bfe14642)

 
## Additional context
 
- **Checkboxes**: Used for selecting ENC items, marked with the `admiralty-checkbox` class and containing the `aria-label` attribute despite having a `role="generic"`.
- **File Upload Input**: An `input` element for file uploads has the `aria-label` attribute despite being assigned the `generic` role.

**Comment by developernm** [comment]
The user adding the `aria-label` tag to the host element being `<admiralty-checkbox>` is causing the error "The 'aria-label' attribute is being used on elements with the generic role". 

As the component already contains a label nested within, the users need to add an `aria-label` tag is redundant. 

The suggestion of implementing logic to have the label hidden but still visible for screen readers is a possibility, but wouldn't stop the user from getting this error, as they've added the property themselves.

**Comment by developernm** [comment]
https://github.com/UKHO/admiralty-design-system/pull/269
